doc/classes/TLSOptions.xml

```diff
@@ -29,15 +29,15 @@
 			<description>
 				Creates a TLS client configuration which validates certificates and their common names (fully qualified domain names).
 				You can specify a custom [param trusted_chain] of certification authorities (the default CA list will be used if [code]null[/code]), and optionally provide a [param common_name_override] if you expect the certificate to have a common name other then the server FQDN.
-				Note: On the Web plafrom, TLS verification is always enforced against the CA list of the web browser. This is considered a security feature.
+				Note: On the Web platform, TLS verification is always enforced against the CA list of the web browser. This is considered a security feature.
```

Comment:
While at it, it would be good to fix the style of "Note:" to be consistent with the rest of the docs. It should be "[b]Note:[/b] On the Web platform, ...".